{
    "MITRE": {
        "CVE_data_meta": {
            "ASSIGNER": "cve@gitlab.com",
            "ID": "CVE-2021-22229",
            "STATE": "PUBLIC"
        },
        "affects": {
            "vendor": {
                "vendor_data": [
                    {
                        "product": {
                            "product_data": [
                                {
                                    "product_name": "GitLab",
                                    "version": {
                                        "version_data": [
                                            {
                                                "version_value": ">=12.8, <13.11.6"
                                            },
                                            {
                                                "version_value": ">=13.12, <13.12.6"
                                            },
                                            {
                                                "version_value": ">=14.0, <14.0.2"
                                            }
                                        ]
                                    }
                                }
                            ]
                        },
                        "vendor_name": "GitLab"
                    }
                ]
            }
        },
        "credit": [
            {
                "lang": "eng",
                "value": "This vulnerability has been discovered internally by the GitLab team"
            }
        ],
        "data_format": "MITRE",
        "data_type": "CVE",
        "data_version": "4.0",
        "description": {
            "description_data": [
                {
                    "lang": "eng",
                    "value": "An issue has been discovered in GitLab CE/EE affecting all versions starting with 12.8. Under a special condition it was possible to access data of an internal repository through project fork done by a project member."
                }
            ]
        },
        "impact": {
            "cvss": {
                "attackComplexity": "HIGH",
                "attackVector": "NETWORK",
                "availabilityImpact": "NONE",
                "baseScore": 5.8,
                "baseSeverity": "MEDIUM",
                "confidentialityImpact": "HIGH",
                "integrityImpact": "NONE",
                "privilegesRequired": "NONE",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N",
                "version": "3.1"
            }
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": [
                        {
                            "lang": "eng",
                            "value": "Improper access control in GitLab"
                        }
                    ]
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "name": "https://gitlab.com/gitlab-org/gitlab/-/issues/332609",
                    "refsource": "MISC",
                    "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/332609"
                },
                {
                    "name": "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22229.json",
                    "refsource": "CONFIRM",
                    "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22229.json"
                }
            ]
        }
    },
    "NVD": {
        "configurations": {
            "CVE_data_version": "4.0",
            "nodes": []
        },
        "cve": {
            "CVE_data_meta": {
                "ASSIGNER": "cve@gitlab.com",
                "ID": "CVE-2021-22229"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "en",
                        "value": "An issue has been discovered in GitLab CE/EE affecting all versions starting with 12.8. Under a special condition it was possible to access data of an internal repository through project fork done by a project member."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": []
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://gitlab.com/gitlab-org/gitlab/-/issues/332609",
                        "refsource": "MISC",
                        "tags": [],
                        "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/332609"
                    },
                    {
                        "name": "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22229.json",
                        "refsource": "CONFIRM",
                        "tags": [],
                        "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2021/CVE-2021-22229.json"
                    }
                ]
            }
        },
        "impact": {},
        "lastModifiedDate": "2021-07-06T21:15Z",
        "publishedDate": "2021-07-06T21:15Z"
    }
}